rgw/v2/tests/s3_swift/test_bucket_notifications.py

```diff
@@ -236,16 +254,22 @@ def test_exec(config, ssh_con):
                         bucket_name_to_create,
                     )
 
-        # delete topic logs on kafka broker
-        notification.del_topic_from_kafka_broker(topic_name)
+                # delete topic logs on kafka broker
+                notification.del_topic_from_kafka_broker(topic_name)
 
     # check sync status if a multisite cluster
     reusable.check_sync_status()
 
     # check for any crashes during the execution
     crash_info = reusable.check_for_crash()
-    if crash_info:
-        raise TestExecError("ceph daemon crash found!")
+    if config.user_remove:
+        if crash_info:
+            raise TestExecError("ceph daemon crash found!")
+        for i in all_users_info:
```

Comment:
@anrao19 , could we have a check on user remove after checking the crash [ at line 268]